packages/components/src/Enumeration/Items/Item/Item.component.js

```diff
@@ -32,11 +32,8 @@ function itemDefaultActionsClasses() {
 }
 
 function Item({ id, item, searchCriteria, showCheckboxes }) {
-	const {
-		key,
-		actions,
-		onSelectItem,
-	} = item.itemProps;
+	const { key, actions, onSelectItem } = item.itemProps;
+	const actualLabel = item[key] instanceof Array ? item[key].join(',') : item[key];
```

Comment:
What do you mean by `actualLabel`?

Reply:
We use this variable two times in Item component, in both cases it is text inside button, aka button label, and it also depends on what will be in "item[key]", whether array or a string
So, because of this dependance I decided to name it like this (I also was thinking about joinedLabel)
How would You propose to name it?